nextflow.config

```diff
@@ -73,24 +73,25 @@ params {
     sentieon_haplotyper_emit_mode = "variant" // default value for Sentieon haplotyper
 
     // Annotation
-    vep_out_format      = "vcf"
-    vep_custom_args     = "--everything --filter_common --per_gene --total_length --offline --format vcf" // Default arguments for VEP
-    vep_dbnsfp          = null  // dbnsfp plugin disabled within VEP
-    dbnsfp              = null  // No dbnsfp processed file
-    dbnsfp_tbi          = null  // No dbnsfp processed file index
-    dbnsfp_consequence  = null  // No default consequence for dbnsfp plugin
-    dbnsfp_fields       = "rs_dbSNP,HGVSc_VEP,HGVSp_VEP,1000Gp3_EAS_AF,1000Gp3_AMR_AF,LRT_score,GERP++_RS,gnomAD_exomes_AF" // Default fields for dbnsfp plugin
-    vep_loftee          = null  // loftee plugin disabled within VEP
-    vep_spliceai        = null  // spliceai plugin disabled within VEP
-    spliceai_snv        = null  // No spliceai_snv file
-    spliceai_snv_tbi    = null  // No spliceai_snv file index
-    spliceai_indel      = null  // No spliceai_indel file
-    spliceai_indel_tbi  = null  // No spliceai_indel file index
-    vep_spliceregion    = null  // spliceregion plugin disabled within VEP
-    outdir_cache        = null  // No output directory for cache
-    snpeff_cache        = null  // No directory for snpEff cache
-    vep_cache           = null  // No directory for VEP cache
-    vep_include_fasta   = false // Don't use fasta file for annotation with VEP
+    dbnsfp                    = null // No dbnsfp processed file
+    dbnsfp_consequence        = null // No default consequence for dbnsfp plugin
+    dbnsfp_fields             = "rs_dbSNP,HGVSc_VEP,HGVSp_VEP,1000Gp3_EAS_AF,1000Gp3_AMR_AF,LRT_score,GERP++_RS,gnomAD_exomes_AF" // Default fields for dbnsfp plugin
+    dbnsfp_tbi                = null // No dbnsfp processed file index
+    snpeff_cache              = 's3://annotation-cache/snpeff_cache/'
+    spliceai_indel            = null // No spliceai_indel file
+    spliceai_indel_tbi        = null // No spliceai_indel file index
+    spliceai_snv              = null // No spliceai_snv file
+    spliceai_snv_tbi          = null // No spliceai_snv file index
+    use_annotation_cache_keys = true
+    vep_cache                 = 's3://annotation-cache/vep_cache/'
+    vep_cache                 = null // No directory for VEP cache
```

Comment:
Is there a bug around here?  `vep_cache` is being set twice in a row.

Reply:
typo, good catch